charts/delphai-deployment/values.yaml

```diff
@@ -135,6 +135,15 @@ ingress:
   # Run Cloudflare purge_cache after deployment
   purgeCloudflareCache: false
 
+rabbitmq:
+  enable: false
+  
+  # The name of the RabbitMQ queue that this service consumes
+  queueName: "service.{{ $.Release.Namespace }}"
```

Comment:
The GitHub Actions workflow takes this string from the GitHub repo name and uses it consistently all through the deployment script.

Reply:
Are variables in `values.yaml` expanded? I'm sure they aren't

Reply:
A missing `tpl` call added in b23fb45467f94bb95eda145e607c16d332f542e7